src/measure_theory/ae_eq_fun.lean

```diff
@@ -23,12 +84,14 @@ variables [measurable_space β]
 
 variables (α β)
 
+/-- The equivalence relation of being almost everywhere equal -/
 instance ae_eq_fun.setoid : setoid { f : α → β // measurable f } :=
 ⟨ λf g, ∀ₘ a, f.1 a = g.1 a,
   assume ⟨f, hf⟩, by filter_upwards [] assume a, rfl,
   assume ⟨f, hf⟩ ⟨g, hg⟩ hfg, by filter_upwards [hfg] assume a, eq.symm,
   assume ⟨f, hf⟩ ⟨g, hg⟩ ⟨h, hh⟩ hfg hgh, by filter_upwards [hfg, hgh] assume a, eq.trans ⟩
 
+/-- Quotient space by the above equivalence relation. -/
```

Comment:
A docstring is designed to be seen as a tooltip in another file, when one writes `eq_eq_fun` and puts the cursor on it. So, avoid "the above equivalence relation" as it would not make sense in this context.